ouroboros-consensus/src/Ouroboros/Consensus/Node/Serialisation.hs

```diff
@@ -149,10 +148,3 @@ instance SerialiseNodeToNode blk (GenTxId     blk)
       encodeNodeToNode cfg version h
   decodeNodeToNode cfg version =
       WrapGenTxId <$> decodeNodeToNode cfg version
-
-instance SerialiseNodeToClient blk (ApplyTxErr     blk)
```

Comment:
I'm not exactly sure why I need to remove this. But it seemed like the cleanest option. I'd like to discuss it on a call with someone. The motivations are some combination of:

  - `SerialiseNodeToClient` being a multi-parameter class
  - Certain types (eg the new `AlonzoApplyTxError`) being defined "above" the scope of `ShelleyBlock`
  - `Ouroboros.Consensus.Node.Run.SerialiseNodeToClientConstraints blk` requiring `SerialiseNodeToClient blk (ApplyTxErr blk)`
  - Wanting a single `era`-polymorphic instance of `SerialiseNodeToClient (ShelleyBlock era)` and hence `SerialiseNodeToClientConstraints (ShelleyBlock era)`
  - No longer having an `era`-polymorphic instance of `ApplyTxErr (ShelleyBlock era)`; we sort of have that, but it just defers to another type family for whom we do not have a single `era`-polymorphic instance.

Reply:
Removing this instance is why I have to adjust other `... ApplyTxErr blk ...` constraints/instances to explicitly include the `WrapApplyTxErr` throughout the rest of this PR.

Reply:
Outdated -- PR input-output-hk/cardano-ledger-specs#2379 supplants this